corehq/apps/cloudcare/views.py

```diff
@@ -220,7 +221,26 @@ def form_context(request, domain, app_id, module_id, form_id):
 cloudcare_api = login_or_digest_ex(allow_cc_users=True)
 
 
+def get_cases_vary_on(request, domain):
+    return [
+        request.couch_user.get_id
+        if request.couch_user.is_commcare_user() else request.REQUEST.get('user_id', ''),
+        request.REQUEST.get('ids_only', 'false'),
+        request.REQUEST.get('case_id', ''),
+        request.REQUEST.get('footprint', 'false'),
+        request.REQUEST.get('include_children', 'false'),
+        request.REQUEST.get('closed', 'false'),
+        json.dumps(get_filters_from_request(request)),
+        domain,
+    ]
+
+
+def get_cases_skip_arg(request, domain):
+    return string_to_boolean(request.REQUEST.get('cache_bust', 'true'))
```

Comment:
Did you also only want to cache when `ids_only=True`? otherwise I'm not sure what would prevent this from caching the full results not just the ID lits.